packages/react/src/component-tests/IcTextField/IcTextFieldTestData.tsx

```diff
@@ -31,3 +34,362 @@ export const Uncontrolled = () => {
     />
   );
 };
+
+export const SimpleTextField = (): ReactElement => (
+  <div style={style}>
+    <IcTextField label="Test TextField Label"></IcTextField>
+  </div>
+);
+
+export const TextFieldWithHelperTextPlaceholderIconRequiredMaxlength =
+  (): ReactElement => (
+    <div style={style}>
+      <IcTextField
+        maxLength={25}
+        label="What is your favourite coffee?"
+        required
+        placeholder="Please enter…"
+        helperText="Such as Arabica, Robusta or Liberica"
+      >
+        <SlottedSVG
```

Comment:
Not at all necessary for approval on this PR, but I think you can extract the SlottedSVG into it's own variable and paste than it for each example in the file, to cut down on duplicated markup :)

Reply:
Check out an example in IcSideNavigationTestData - the variable ReusableSlottedIcon